scp/lib/drawer_screens/dev_info.dart

```diff
@@ -132,6 +137,7 @@ class DevInfo extends StatelessWidget {
   Widget contactCard(BuildContext context, String name, String contact) {
     //var queryWidth = MediaQuery.of(context).size.width;
     var textScaleFactor = MediaQuery.of(context).textScaleFactor;
+    var width = MediaQuery.of(context).size.width;
```

Comment:
Instead of writing this in every file, make a size config class and initialise it with the app and use it wherever required. You can refer to project-maple/elaichi for this